auth/src/main/java/org/aerogear/mobile/security/impl/SecurityCheckExecutorImpl.java

```diff
@@ -0,0 +1,73 @@
+package org.aerogear.mobile.security.impl;
+
+import android.content.Context;
+
+import org.aerogear.mobile.core.metrics.MetricsService;
+import org.aerogear.mobile.security.Check;
+import org.aerogear.mobile.security.SecurityCheck;
+import org.aerogear.mobile.security.SecurityCheckExecutor;
+import org.aerogear.mobile.security.SecurityCheckResult;
+import org.aerogear.mobile.security.metrics.SecurityCheckResultMetric;
+
+import java.util.ArrayList;
+
+public class SecurityCheckExecutorImpl implements SecurityCheckExecutor {
+    private final ArrayList<SecurityCheck> checks;
+    private final Context context;
+
+    private MetricsService metricsService;
+
+    public SecurityCheckExecutorImpl(Context context) {
+        this.checks = new ArrayList<>();
+        this.context = context;
+    }
+
+    @Override
+    public SecurityCheckExecutor addCheck(Check check) {
```

Comment:
I think that we should add 

`public SecurityCheckExecutor addCheck(SecurityCheck check) {`

too. That way the user will be able to add custom checks as well